extensions/applicationinsights-debugplugin-js/scripts/publishReleaseToCdn.ps1

```diff
@@ -0,0 +1,536 @@
+param (
+    [string] $releaseFrom = $null,                      # The root path for where to find the files to be released
```

Comment:
Also mostly a copy from the AISKU -- changed to support only the ext/ folder and ai.dbg. prefix